services/navigation/builtin/builtin.go

```diff
@@ -543,7 +543,7 @@ func (svc *builtIn) moveToWaypoint(ctx context.Context, wp navigation.Waypoint,
 	}
 	cancelCtx, cancelFn := context.WithCancel(ctx)
 	defer cancelFn()
-	executionID, err := svc.motionService.MoveOnGlobeNew(cancelCtx, req)
+	executionID, err := svc.motionService.MoveOnGlobe(cancelCtx, req)
```

Comment:
lets also delete the helper function `pollUntilMOGSuccessOrError` in favor of the one we wrote and tested in the motion package

Reply:
good catch

Reply:
fixed